workflow/src/main/kotlin/com/r3/corda/sdk/token/workflow/flows/MoveToken.kt

```diff
@@ -40,7 +41,11 @@ object MoveToken {
             val owningParty: AbstractParty = if (anonymous) {
                 subFlow(RequestConfidentialIdentity.Initiator(ownerSession)).party.anonymise()
             } else owner
-
+    
+            if (ownedToken is TokenPointer<*>) {
```

Comment:
I'm not sure how this will work. I set up the distribution list so that only the creator of the evolvable token type has the distribution list, as they are the only ones which can update the list. Here we are setting up a distribution list on the party which is moving the token (which references the evolvable token type) to another node. So they will have a list but there is no code path in the token SDK currently which would result in this non-issuer party forwarding on the evolvable token updates.... This is where data distribution groups come in. I realise that for now, the functionality here is quite limited until we get data distribution groups. The solution is for all `Party`s to manually sign up with the maintainer of the evolvable token type using the `RequestAdditionToDistributionList` flow.

Reply:
Shall i remove this AddPartyToDistributionList?  I misunderstood the concept and to enable all parties receiving the updates when an update happens on the evolvable token, I added this.